### ISSUE TITLE
Use PostCSS 6.0

### ISSUE THREAD

**Issue by ai** [postcss/postcss-cli#125]
[PostCSS 6.0](https://github.com/postcss/postcss/releases/tag/6.0.0) was released. API wasn’t changed dramatically, so you need only update dependency and release new version.

@RyanZim write me if you need any help.

**Comment by RyanZim** [comment]
@ai Looks pretty easy. I assume this should be semver-major for postcss-cli.

**Comment by SerkanSipahi** [comment]
@RyanZim Please deploy new version. I get erros when working with postcss-cli
```error
Your current PostCSS version is 5.2.17, but postcss-nested uses 6.0.1. Perhaps this is the source of the error below.
```

**Comment by michael-ciniawsky** [comment]
@SerkanSipahi Pin `postcss-nested` to `^1.x` semver range in the meantime, `2.x`works the same but with PostCSS 6 API

**Comment by SerkanSipahi** [comment]
@RyanZim @michael-ciniawsky @ai please let me know if postcss-cli are ready for PostCSS 6.0, thnx.

**Comment by RyanZim** [comment]
Released in v4.0.0: https://github.com/postcss/postcss-cli/releases/tag/v4.0.0 :tada:

**Comment by SerkanSipahi** [comment]
@RyanZim thanx :1st_place_medal:
